src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -115,6 +115,13 @@
       + "is automatically destroyed.";
   public static final String CONSUMER_INSTANCE_TIMEOUT_MS_DEFAULT = "300000";
 
+  public static final String SIMPLE_CONSUMER_POOL_SIZE_CONFIG = "simpleconsumer.pool.size";
+  private static final String
+      SIMPLE_CONSUMER_POOL_SIZE_DOC =
+      "maximum number of SimpleConsumers that can be instantiated by broker."
+      + " If empty, then no pool is used.";
+  public static final String SIMPLE_CONSUMER_POOL_SIZE_DEFAULT = "3";
```

Comment:
Any reasoning behind this default? Default values are pretty important, so I'd like to know if this is actually a reasonable number. If a single API request needs to make multiple requests to the broker, we could end up using it for quite awhile, right? Which would then block or cause errors all but three requests. Since simple consumers should be pretty cheap, maybe a bigger value would make sense here?

Also, since this really is the max number, the setting should probably be simpleconsumer.pool.size.max